assets/js/stripe-admin.js

```diff
@@ -91,10 +91,10 @@ jQuery( function( $ ) {
 				e.preventDefault();
 
 				// Remove any previous messages.
-				$( '.wc-stripe-apple-pay-domain-message' ).remove();
+				$( '.wc-stripe-apple-pay-domain-message' ).text( '' );
 
 				if ( ! wc_stripe_admin.getSecretKey() ) {
-					$( '#wc-gateway-stripe-apple-pay-domain' ).after( '<p class="wc-stripe-apple-pay-domain-message" style="color:red;">' + wc_stripe_admin_params.localized_messages.missing_secret_key + '</p>' );
+					$( '.wc-stripe-apple-pay-domain-message' ).attr( 'style', 'color: red;' ).text( wc_stripe_admin_params.localized_messages.missing_secret_key );
```

Comment:
If you want to go this route with the styles, you might as well use `css()` method.